src/components/Atoms/AddTaskButton/index.jsx

```diff
@@ -0,0 +1,42 @@
+import plus from "../../../assets/svg/plus.svg";
+import styled from "styled-components";
+import React from "react";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import FONT_FAMILY from "../../../variables/font_family";
+
+export const AddTaskButton = ({ onClick }) => {
+  return (
+    <StyledAddTaskButton onClick={onClick}>
+      <StyledImg src={plus} />
+      <StyledButton>タスクを追加</StyledButton>
+    </StyledAddTaskButton>
+  );
+};
+
+const StyledAddTaskButton = styled.button`
+  border: none;
+  outline: none;
+  cursor: pointer;
+  background: none;
+  transition: 0.2s ease;
+  display: flex;
+  align-items: center;
+  padding: 2px 6px;
+  height: 24px;
+  gap: 10px;
+  :hover {
+    background-color: ${COLOR.GREEN_TRANSLUCENT};
+    border-radius: 12px;
+  }
+`;
+const StyledImg = styled.img`
+  width: 20px;
+  height: 20px;
+`;
+
+const StyledButton = styled.div`
```

Comment:
```suggestion
const Text = styled.span`
```

個人的な意見にはなりますが、`StyledButton` というコンポーネント名だと `button` 要素であるかのような印象を受けます。
実際にはこのコンポーネントは「タスクを追加」という文言を表示するものになるので、このようなコンポーネント名にしたうえで `span` 要素にしてあげるのが適切可と思います。

`Styled` という命名については、自分は特に拘る必要はないのではと思っています。
（他に適切な命名が思いつかなかった場合に仕方なく使うイメージ）

ただこのへんは最終的には個人の好みなので、コンポーネント名が原因でレビュー不通過になることはないかと思われます。参考までにお願いします。